tests/Search20Test.php

```diff
@@ -45,39 +45,39 @@ public function testSimpleSearchResultValidatesAgainstSchema(): void
 
     public function testSimpleSearchResultContentsConsistent()
     {
-      $json = $this->s->GetSearchMatches(null, 'khmer', 1);
+      $json = $this->s->GetSearchMatches(null, 'khmer', 1, 1);
       $json = json_decode(json_encode($json));
       $this->schema->in($json);
       $this->assertJsonStringEqualsJsonFile(__DIR__ . '/fixtures/Search.2.0.khmer.json', json_encode($json), "Search for 'khmer' gives same results as Search.2.0.khmer.json");
     }
 
     public function testPhraseSearchResult()
     {
-      $json = $this->s->GetSearchMatches(null, 'khmer angkor', 1);
+      $json = $this->s->GetSearchMatches(null, 'khmer angkor', 1, 1);
```

Comment:
Do we have any tests where the new `obsolete` parameter is `0`?

Reply:
Not yet; I can add one.